server/models/outlook.js

```diff
@@ -122,13 +119,15 @@ class Outlook {
     */
 
     const issueDate = new Date(outlook.issued_at)
-    const date = issueDate.getDate()
+    // const date = issueDate.getDate() // DL: incorrect dates in model
```

Comment:
For lines 122-130. Why are the lines of code commented out? Shouldn't leave unnecessarily commented out code. But it looks like the new lines do pretty much the same as the commented out